### PR TITLE
feat: complete pending introduce protocol methods

### PR DESCRIPTION
**Pull request by ToJen**
Signed-off-by: Tomisin Jenrola <tomisin.jenrola@securekey.com>

**Title:**
Added all methods for both implementations of the Introduce protocol

**Description:**
Part of #1958 

Summary:
- updated the local and remote agent implementations
- updated the README
- added an Apache 2.0 license to the mobile bindings subrepo
- wrote necessary unit tests

### CODE REVIEW

File: cmd/aries-agent-mobile/pkg/wrappers/constants.go [hunk 1]
```diff
@@ -12,4 +12,31 @@ const (
 
 	// HandlerActions is the name of the actions operation
 	HandlerActions = "Actions"
+
+	// HandlerSendProposal is the name of the send proposal operation
+	HandlerSendProposal = "SendProposal"
+
+	// HandlerSendProposalWithOOBRequest is the name of the send proposal with out-of-bound request operation
+	HandlerSendProposalWithOOBRequest = "SendProposalWithOOBRequest"
```
Comment (soluchok) [inline cmd/aries-agent-mobile/pkg/wrappers/constants.go:20]: I think it makes sense to expose constants from commang.go file and use them here.

Comment (ToJen) [reply cmd/aries-agent-mobile/pkg/wrappers/constants.go:20]: i was wondering where to put these. they're used by both command and rest packages which are independent of each other. is it okay to move them to `command/command.go` and use them in `rest/*.go` as `command.HandlerX`?

Comment (soluchok) [reply cmd/aries-agent-mobile/pkg/wrappers/constants.go:20]: I meant to expose them in `pkg/controller/command/introduce/command.go`. And then reuse them from Aries.

File: cmd/aries-agent-mobile/README.md [hunk 2]
```diff
@@ -103,6 +103,12 @@ This is an example of how the imported framework can be used:
     NSString *actionsResp = resp.actionsResponse;
 ```
 
+
+### c. Demo apps
+
+For examples of mobile apps built with the aires-agent-mobile bindings, see [trustbloc/aries-examples](https://github.com/trustbloc/aries-examples).
```
Comment (llorllale) [inline cmd/aries-agent-mobile/README.md:109]: aries-agent-mobile

File: cmd/aries-agent-mobile/pkg/wrappers/constants.go (hunk 1 above)
Comment (llorllale) [reply cmd/aries-agent-mobile/pkg/wrappers/constants.go:20]: @soluchok you mean a new file `pkg/controller/command/commands.go` ? Agreed

File: cmd/aries-agent-mobile/README.md (hunk 2 above)
Comment (ToJen) [reply cmd/aries-agent-mobile/README.md:109]: whoops

File: cmd/aries-agent-mobile/pkg/wrappers/constants.go (hunk 1 above)
Comment (ToJen) [reply cmd/aries-agent-mobile/pkg/wrappers/constants.go:20]: maybe expose these: https://github.com/hyperledger/aries-framework-go/blob/master/pkg/controller/command/introduce/command.go#L58-L67

Comment (ToJen) [reply cmd/aries-agent-mobile/pkg/wrappers/constants.go:20]: and these: https://github.com/hyperledger/aries-framework-go/blob/master/pkg/controller/rest/introduce/operation.go#L28-L38

Comment (llorllale) [reply cmd/aries-agent-mobile/pkg/wrappers/constants.go:20]: @ToJen yes

**Pull request merged**